src/components/button/button.d.ts

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+export interface ButtonProps {
+  as?: 'primary' | 'secondary';
+  disabled?: boolean;
+  theme?: 'blue' | 'grey' | 'magenta' | 'magenta-dull' | 'red' | 'white';
+  size?: 'small' | 'medium' | 'large';
+  subtext?: any;
```

Comment:
I wonder if `any` type is a good practice.

Reply:
Yeah I've missed that one, I'm going to set it to string as that's what it is in the propTypes